enhancements/cluster-scope-secret-volumes/csi-driver-host-injections.md

```diff
@@ -213,118 +163,94 @@ the underlying, associated Pod references CSI volumes using `Shares`
-associated `ServiceAccount` for the Pod has sufficient access to the `ProjectedResource`
-- if sufficient access does not exist, the admission plugin marks the Pod as invalid/forbidden.
-- if sufficient access does exists, the admission plugin adds a CSI volume into the Pod's `volume` array, to match the
-example noted above.
-
+  ```yaml
+  apiVersion: v1
+  kind: Pod
+  metadata:
+    name: some-pod
+  spec:
+    containers:
+      ...
+      volumeMounts:
+      - name: shared-item-name
+        mountPath: /path/to/shared-item-data
+    volumes:
+    - name: entitlement
+      csi:
+        driver: projectedresources.storage.openshift.io
+        volumeAttributes:
+          share: the-projected-resource
+  ```
 
 The CSI Driver, aside from all the CSI and filesystem bits (see below for details), performs:
-- reads of the `ProjectedResource` object referenced in the `volumeMetadata` which is available in the `NodePublishVolumeRequest`
-that comes into `NodePublishVolume`
-- the name, namespace, and `ServiceAccount` of the Pod wanting the volume is also included by the kubelet in the
-`NodePublishVolumeRequest` under the keys `csi.storage.k8s.io/pod.name`, `csi.storage.k8s.io/pod.namespace`,  and
-`csi.storage.k8s.io/serviceAccount.name` in the map returned by the `GetVolumeContext()` call
+
+- reads of the `ProjectedResource` object referenced in the `volumeMetadata` which is available in the `NodePublishVolumeRequest` that comes into `NodePublishVolume`.
+- the name, namespace, and `ServiceAccount` of the Pod wanting the volume is also included by the kubelet in the `NodePublishVolumeRequest` under the keys `csi.storage.k8s.io/pod.name`, `csi,storage.k8s.io/pod.namespace`, and `csi.storage.k8s.io/serviceAccount.name` in the map returned by the `GetVolumeContext()` call.
 - the `podInfoOnMount` field of the CSI Driver spec triggers the setting of these values; here is
 [an example from the hostpath plugin](https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/deploy/kubernetes-1.17/hostpath/csi-hostpath-driverinfo.yaml#L12)
-- with all this necessary metadata, the CSI driver then
-- reads of the `Secret` or `ConfigMap` noted in the `ProjectedResource`, storing it on disk on initial
-reference to said `Secret` or `ConfigMap`
-- creation of a tmpfs filesystem for each requesting `ServiceAcount` that contains the `Secret` / `ConfigMap` data
-- creation of k8s watches `ProjectedResources`, and for `Secret` / `ConfigMaps` in the namespaces listed in each discovered
-`ProjectedResources`
-- final returning of a volume that is based off of the `ServiceAccount` specific tmpfs for the data noted in the
-`volumeAttributes`
+
+With all this necessary metadata, the CSI driver then:
+
+- reads of the `Secret` or `ConfigMap` noted in the `ProjectedResource`, storing it on disk on initial reference to said `Secret` or `ConfigMap`.
```

Comment:
a failure in the prior form of the EP @adambkaplan that we can correct since you are reworking.... SAR checks are performed on the initial provisioning as well

Some brief detail on the SAR check seems useful as well, and should be included:  namely the SAR check sees if the serviceaccount for the Pod to be provisioned has permission to get the referenced Share